### ISSUE TITLE
FORMAT_HUMAN does not do anything

### ISSUE THREAD

**Issue by jerry1970** [Ayesh/php-timer#2]
Hi, the `FORMAT_HUMAN` constant does not do anything. I guess you need another `case` in the `formatTime` method.

**Comment by atymic** [comment]
Closed by #6 🎉 @Ayesh

**Comment by atymic** [comment]
@Ayesh 

Can you tag a new release for this please? 🙏

**Comment by Ayesh** [comment]
Sorry about the delay. I will work on some cleanup and cut a release soon this weekend. Thank you for your patience.

**Comment by atymic** [comment]
No worries :)

**Comment by atymic** [comment]
Any updates @Ayesh? Got it pinned to my fork atm but would like to move back to mainline :)

**Comment by Ayesh** [comment]
Thanks a lot for your patience @atymic and @jerry1970 . I wanted to implement the a wider spectrum of time ranges (5ms, 5 sec, 5 min, 5 hours, 5 days, 5 weeks, 5 months, to 5 years). I have just released v2.1.1 with this feature, thanks to @atymic's initial implementation. I changed the wording "1 second" to "1 sec" to align with "1 min" pattern. 

Once again thanks a lot for your patience :) If you like any further changes, I'd be happy to work with you. 

Cheers.
